inline-js/package.yaml

```diff
@@ -47,3 +53,21 @@ tests:
     ghc-options: -threaded -with-rtsopts=-N
     dependencies:
       - inline-js
+
+tests:
+  inline-js-test-suite:
+    source-dirs: tests
+    main: Main.hs
+    ghc-options:
+    - -Wall
```

Comment:
One minor suggestion: can we increase 1 level of indentation here for visual comfort?